### DISCUSSION TITLE
[#1065642] Automatic Crouch while in no-clip

### DISCUSSION THREAD

**Discussion by FrederoxDev** [Mojang/minecraft-editor#124] [Bugs]
To reproduce enter crosshair mode and fly with a block above the player:
![image](https://github.com/Mojang/minecraft-editor/assets/69014593/438b34b3-ab36-464a-90fc-f271653d1d69)

This isn't ideal as it will cause the camera to shift in first person mode and can happen frequently when moving around

**Comment by minecraftchloe** [comment]
Thank you!
